src/common/json_output.rs

```diff
@@ -35,20 +39,27 @@ pub struct EnclaveDescribeInfo {
     #[serde(rename = "Flags")]
     /// The bit-mask which provides the enclave's launch flags.
     pub flags: String,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(flatten)]
+    /// Build measurements containing PCRs
+    pub build_info: Option<EnclaveBuildInfo>,
```

Comment:
This seems to be part of another patch.

Reply:
> How backwards compatible is this change? If I start an enclave on the current published version and then upgrade the tooling to this, will it still work, or will something crash?

I tested backwards compatibility and developed the feature in this way. The output of the current published version will be missing the additional field of the describe output which will be interpreted by the new CLI process as being `None`, so nothing crashes.